js-advance/js-modern-syntax_destructuring-objects/index.js

```diff
@@ -40,3 +48,10 @@ const person = {
   age: 24,
   firstName: "Alex",
 };
+
+export const { lastName: personLastName } = person;
+
+export const moreInformation = {
```

Comment:
you can do all in one line here: 

export const { lastName: personLastName, ...moreInformation } = person;